Src/SwqlStudio/QueryTab.cs

```diff
@@ -476,9 +479,9 @@ private void queryWorker_RunWorkerCompleted(object sender, System.ComponentModel
             }
         }
 
-        private static void AutoResizeColumns(DataGridView grid)
+        private void AutoResizeColumns(DataGridView grid)
         {
-            const int maxSize = 200;
+            int maxSize = base.LogicalToDeviceUnits(200);
```

Comment:
While not really needed, I wanted to keep the column layout close to the original, if possible. `LogicalToDeviceUnits` takes a number of pixels in 100% DPI and returns how many pixels it would be using the current DPI settings.